client/components/domains/domain-registration-suggestion/utility.js

```diff
@@ -54,6 +54,20 @@ const newTlds = [
 
 const testTlds = [ '.de' ];
 
+export function getTld( domainName = '' ) {
+	return last( domainName.split( '.' ) );
```

Comment:
I have concerns about this TLD extraction function. Do we normally split by a comma and remove the first item in the array instead?

Reply:
Replaced this function with `getTld` from [lib/domains](https://github.com/Automattic/wp-calypso/blob/master/client/lib/domains/index.js#L233).